src/arm/linux/aarch64-isa.c

```diff
@@ -151,4 +156,15 @@ void cpuinfo_arm64_linux_decode_isa_from_proc_cpuinfo(
 	if (features & CPUINFO_ARM_LINUX_FEATURE_ASIMDFHM) {
 		isa->fhm = true;
 	}
+
+    #define PR_SVE_GET_VL 51
+    #define PR_SVE_VL_LEN_MASK 0xffff
```

Comment:
Don't you want to add something like `#ifndef PR_SVE_GET_VL` guard here? And also, please mention which system headers they should be defined in

Reply:
Done.